modules/apps/transfer/keeper/relay_test.go

```diff
@@ -149,16 +157,92 @@ func (suite *KeeperTestSuite) TestSendTransfer() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestSendTransferDoesNotSetTotalEscrowAmountForSourceIBCToken() {
+	/*
+		Given the following flow of tokens:
+
+		chain A (channel 0) -> (channel-0) chain B (channel-1) -> (channel-1) chain A
```

Comment:
nit: wouldn't this be channel-0?
```suggestion
		chain A (channel 0) -> (channel-0) chain B (channel-1) -> (channel-0) chain A
```